lib/rules/no-trailing-zero.js

```diff
@@ -2,7 +2,7 @@
 
 var helpers = require('../helpers');
 
-var trailingZeroRegex = /^(\d+\.|\.)+(\d*?[1-9])0+$/;
+var trailingZeroRegex = /^(\d+\.|\.)+(\d*?)0+$/;
```

Comment:
I believe `^(\d+\.|\.)` could be changed to `^(\d*\.)` which in my opinion is a bit easier to read.